src/azure-cli/azure/cli/command_modules/acs/_resourcegroup.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.azclierror import AzCLIError
+
+from ._client_factory import cf_resource_groups
+
+
+def get_rg_location(ctx, resource_group_name, subscription_id=None):
```

Comment:
For reference, [this](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L4154) is the corresponding implementation in `custom.py`.